x/params/store/pair.go

```diff
@@ -0,0 +1,15 @@
+package store
+
+// Used for associating paramstore key and field of param structs
+type KeyValuePair struct {
+	Key   []byte
+	Value interface{}
+}
+
+// Slice of KeyFieldPair
+type KeyValuePairs []KeyValuePair
+
+// Interface for structs containing parameters for a module
+type ParamStruct interface {
```

Comment:
s/ParamStruct/ParamSet/g , since it isn't a struct i think we should avoid the Struct suffix.